
#### Error de demora en la conexion ##

#raise ConnectionError(err, request=request)
#requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None))

#######################################


########## Agregando varias veces seguidas al carrito #########

2021-09-11 11:51:08,876 (__init__.py:652 MainThread) ERROR - TeleBot: "A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: message is not modified: specified new message 
content and reply markup are exactly the same as a current content and reply markup of the message"

#################################################################

